src/controllers/shared.controllers/director.nationality.controller.ts

```diff
@@ -120,9 +120,15 @@ const sameNationalityWithChips = (currentOfficerFiling: OfficerFiling, companyAp
     return sameNationality;
   }
 
-  if (currentOfficerFiling.nationality1 === nationality[0]) {
-    if ((!currentOfficerFiling.nationality2 || nationality[1] === currentOfficerFiling.nationality2) &&
-        (!currentOfficerFiling.nationality3 || nationality[2] === currentOfficerFiling.nationality3)) {
+  nationality.forEach((officerNationality, index)=> {
+    if (officerNationality) {
+      nationality[index] = officerNationality.toUpperCase();
+    }
+  });
+
+  if (currentOfficerFiling.nationality1?.toUpperCase() === nationality[0]) {
+    if ((!currentOfficerFiling.nationality2 || nationality[1] === currentOfficerFiling.nationality2.toUpperCase()) &&
```

Comment:
Wouldn't this falsely flag nationalities being the same if nationality2 had been removed/blanked in the filing but was present on CHIPS? In a scenario where dual nationality was being removed for instance